CHANGELOG-developer.next.asciidoc

```diff
@@ -152,6 +152,7 @@ The list below covers the major changes between 7.0.0-rc2 and main only.
 - Add the file path of the instance lock on the error when it's is already locked {pull}33788[33788]
 - Add DropFields processor to js API {pull}33458[33458]
 - Add support for different folders when testing data {pull}34467[34467]
+- Add GCP Carbon Footprint metricbeat data {pull}34820[34820]
```

Comment:
I don't think an entry in this CHANGELOG file is needed as this is more of a user-facing change.